master_xml.py

```diff
@@ -40,8 +40,8 @@ def getRawTooltips(locale):
             continue
         tooltipId = split[1].replace("tooltip_","").replace("_description","").replace("\"", "").lstrip("0")
 
-        # Remove any quotation marks, new lines and html tags.
-        rawTooltips[tooltipId] = cleanHtml(split[2].replace("\"", "").replace("\\n", "\n"))
+        # Remove any quotation marks and new lines.
+        rawTooltips[tooltipId] = split[2].replace("\"", "").replace("\\n", "\n")
```

Comment:
I think `\n` is fine. I don't know of a format that will not accept that as a newline character.

Good catch on the `"` replacement. It was initially there to remove the leading and trailing `"`. I'll rework that.